tools/api-designer/README.md

```diff
@@ -17,27 +17,27 @@ This project depends on other tools in this repo. You have to manually build tho
 
 **Note**: if you make modifications to any of those dependencies and rebuild and the changes don't take effect in the API designer, try:
 
-- uninstalling the dependency from the API designer (remove it from `package.json` and `node_modules` or run `npm uninstall`)
-- adding it back to `package.json`
-- running `npm run build`
+- delete all `dist` folders of the dependencies
+- run `npm run build` again in the `tools` folder
 
 **Troubleshooting**
 
 - If you get build errors on Windows, especially related to typescript types, consider using Powershell instead of CMD
 - [This GitHub thread](https://github.com/parcel-bundler/parcel/issues/7697) was useful in resolving `Cannot resolve 'process'` error that occurred in some environments
 
-**TODO**: Consider organizing this project as a monorepo to make dependency management of related tools easier. The current set up is quick hack.
-
 ### Install dependencies
 
-Run:
+In the `tools` folder (parent of this folder) run:
 
 ```
 npm install
+npm run build
 ```
 
 ### Launch the app locally
 
+In the `tools` folder (parent of this folder) run:
+
 ```
-npm start
+npm run api-designer
```

Comment:
Can we still run from api-designer directory, or must we run from tools?

Reply:
Yes, added alternative way to start the app